src/actions/assets/Protomaps.js

```diff
@@ -0,0 +1,30 @@
+import AssetActionPlugin from "../AssetActionPlugin";
+import URI from 'urijs';
+import i18n from "../../i18n";
+
+// obj & ply files are usually with mime-type text/plain 
+const PROTOMAPS_SUPPORTED_TYPES = [
+  'application/vnd.pmtiles',
+];
+
+export default class Protomaps extends AssetActionPlugin {
+
+  get show() {
+    // Rather check if .pmtiles substring present in this.asset.href or simply this.component.filename.endsWith('pmtiles')
+    return this.component.isBrowserProtocol && (
+      PROTOMAPS_SUPPORTED_TYPES.includes(this.asset.type)
+      || this.asset.href.toLowerCase().includes('.pmtiles')
```

Comment:
I think we should use urijs to get the path component and then do an endsWith check. That will be more accurate.

Reply:
We use urijs everywhere in the code, so to keep it consistent you can use:

```
import URI from 'urijs';
const href = 'http://ABC.com/folderpath/example.pmtiles?v=1';
check = URI(href).suffix() == 'pmtiles'
```